docs/docs/configuration.md

```diff
@@ -52,6 +52,8 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.parquet.bloom-filter-enabled.column.col1       | (not set)                   | Hint to parquet to write a bloom filter for the column: 'col1'                                                                                                                                    |
 | write.parquet.bloom-filter-max-bytes                 | 1048576 (1 MB)              | The maximum number of bytes for a bloom filter bitset                                                                                                                                             |
 | write.parquet.bloom-filter-fpp.column.col1           | 0.01                        | The false positive probability for a bloom filter applied to 'col1' (must > 0.0 and < 1.0)                                                                                                        |
+| write.parquet.stats-enabled.default                  | true                        | Controls whether to collect parquet column statistics when not specified on column level, currently not configurable [parquet-java#3189](https://github.com/apache/parquet-java/issues/3188)      |
```

Comment:
nit: `Default to collect all column statistics in the table. Can disable after [parquet-java#3189](https://github.com/apache/parquet-java/issues/3188)` 

or maybe mention the next parquet release version will be helpful?

Reply:
I changed it to `Controls whether to collect parquet column statistics when not specified on column level, can be configured to false with [parquet-java#3189](https://github.com/apache/parquet-java/issues/3188)`, does it look better?

Reply:
Removed this default config per comments, so it does not apply anymore.